File: patterns/c-patterns/pattern26.c
// Pattern 26: ...

// EEEEE
// *DDDD
// **CCC
// ***BB
// ****A


#include <stdio.h>
int main()
{
    int i, j;
    int n =4;
    for(i =n; i>=0; i--)
    {
        for(j=n; j>=0; j--)
        {
            if(j>i)
            printf("*");
            else
            printf("%c", i+65);
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern26.js
// Pattern 26: ...

// EEEEE
// *DDDD
// **CCC
// ***BB
// ****A

const n = 4;

for (let i = n; i >= 0; i--) {
    let row = '';

    for (let j = n; j >= 0; j--) {
        if (j > i) {
            row += '*';
        } else {
            row += String.fromCharCode(i + 65);
        }
    }

    console.log(row);
}